gcc.symbols.imp

```diff
@@ -134,8 +134,8 @@
   { symbol: [ "sigset_t", private, "<spawn.h>", public ] },
   { symbol: [ "sigset_t", private, "<sys/select.h>", public ] },
   { symbol: [ "sigval", private, "<signal.h>", public ] },
-  { symbol: [ "sockaddr", private, "<sys/socket.h>", private ] },
```

Comment:
Please squash these changes into the commits that introduced these mappings.

Reply:
Same as with struct/union.  I'll do it if you confirm.

Reply:
Yep, would be nice to get this in from the start.